src/shared/copy/notifications.ts

```diff
@@ -1482,9 +1482,9 @@ export const exportAlertToTaskFailure = (
   message: `Failed to create task for ${source} alert. Please check your configuration.`,
 })
 
-export const getResourcesTokensFailure = (): Notification => ({
+export const getResourcesTokensFailure = (tokenType: string): Notification => ({
```

Comment:
maybe provide a default value for `tokenType` like `tokenType: string = 'that token'` or something like that?